src/pages/docs/platform-engineering/managing-runbook-resources.md

```diff
@@ -0,0 +1,96 @@
+    2. Set the `Terraform Backend` field to the [backend](https://developer.hashicorp.com/terraform/language/settings/backends/configuration) configured in the exported module. The step defaults to `s3`, which uses an S3 bucket to store Terraform state. However, any backend provider can be defined here.
+    3. Set the `Octopus Server URL` field to the URL of the Octopus server to export a space from. The default value of `#{Octopus.Web.ServerUri}` references the URL of the current Octopus instance.
+    4. Set the `Octopus API Key` field to the [API key](/docs/octopus-rest-api/how-to-create-an-api-key) used to access the instance defined in the `Octopus Server URL` field.
+    5. Set the `Octopus Space ID` field to the ID of the space to be exported. The default value of `#{Octopus.Space.Id}` references the current space.
+    6. Set the `Octopus Project Name` field to the name of the project that contains the runbook to be exported. The default value of `#{Octopus.Project.Name}` references the current project.
+    7. Set the `Octopus Runbook Name` field to the name of the runbook to serialize.
+    8. Set the `Octopus Upload Space ID` field to the ID of another space to upload the resulting Terraform module zip file to the built-in feed of that space. Leave this field blank to upload the zip file to the built-in feed of the current space.
+
+Executing the runbook will:
+
+* Export the runbook to a Terraform module
+* Zip the resulting files
+* Upload the zip file to the built-in feed of the current space or the space defined in the `Octopus Upload Space ID` field
+
+The zip file has one directory called `space_population` which contains a Terraform module to populate a space with the exported resources.
+
+:::div{.hint}
+Many of the exported resources expose values, like resource names, as Terraform variables with default values. You can override these variables when applying the module to customize the resources, or leave the Terraform variables with their default value to recreate the resources with their original names.
+:::
+
+## Dealing with project variables
+
+The exported module defines only the runbook and the runbook deployment process. It does not define any other project level resources like project variables or library variable sets. Any project that the exported runbook is added to is expected define all the variables referenced by the runbook.
+
+Any project level variables required by the runbook can be defined as Terraform resources and deployed alongside the exported runbook module. The instructions documented in the [Managing project resources](managing-project-resources) section can be used to export a project to a Terraform module. The project level variables can be copied out of the exported project module as needed and placed in their own module as needed. 
```

Comment:
```suggestion
Any project level variables required by the runbook can be defined as Terraform resources and deployed alongside the exported runbook module. The instructions documented in the [Managing project resources](/docs/platform-engineering/managing-project-resources) section can be used to export a project to a Terraform module. The project level variables can be copied out of the exported project module as needed and placed in their own module as needed. 
```

Reply:
:+1: I'll add those suggestions, but leave the link with the short relative url as that is how links in other pages have been defined.